ci-operator/step-registry/ipi/aws/pre/local-zones/byo-vpc/ipi-aws-pre-local-zones-byo-vpc-chain.yaml

```diff
@@ -0,0 +1,28 @@
+chain:
+  as: ipi-aws-pre-local-zones-byo-vpc
```

Comment:
Seem like localzone development was already done in https://github.com/openshift/release/pull/37022, can we use the same provision/pre chain on different cluster profile to run the ci? cc @yunjiang29

Reply:
Yes, this LZ deployment was based on #37022 but will be changed in the next phase (II).
there are two main differences from the goals for this deployment:
1) on phase II (4.14) we will use the CloudFormation template provided on the installer repo, instead of embedding it [it was not added on this PR as we need to run this periodic asap to capture signals of phase I/4.13 deployment]
2) the current LZ deployment is selecting randomly the zone by default, increasing the locations to test. In my last commit, I ported the existing deployment to keep more compatibility [but not yet tested]. The goal is to talk with @yunjiang29 when 1 is finished: https://github.com/openshift/release/compare/e97a4db64b6afac2aaa015ee270ce46def9b5613..e0dbe15a1cce3dee1858ba044b2df4c1fee2746c

Reply:
@mtulio thanks for adding extra test points, I noticed some codes are duplicated as existing code, that would be better if we could re-use them instead of creating new steps.

Reply:
Thanks for your follow up. 

Some background that QE migrate testing from internal jenkins to prow, one important goal is QE and Dev can share the same steps in testing, and do contributions for a unified ci, enhance the collaboration between QE and Dev to work more closer together, also can help keep us on the same page. This is our KR goals. Re-using the same would also help eliminate some cases that someone can reproduce issues, but others can not.

That is why we proposed to re-use the new steps as much as possible.

Reply:
> Thanks for your follow up.
> 
> Some background that QE migrate testing from internal jenkins to prow, one important goal is QE and Dev can share the same steps in testing, and do contributions for a unified ci, enhance the collaboration between QE and Dev to work more closer together, also can help keep us on the same page. This is our KR goals. Re-using the same would also help eliminate some cases that someone can reproduce issues, but others can not.
> 
> That is why we proposed to re-use the new steps as much as possible.

💯 Yeah, that makes total sense to me and I will work to improve that. But I am not sure how to troubleshoot those [failing jobs without access to it](https://github.com/openshift/release/pull/36911#issuecomment-1507927121) - mainly when refactoring/adapting the steps maintained by QE to the use case we are implementing.

I am open to proposals on how to reuse tools that can unblock both teams.

cc @rvanderp3

Reply:
> I am not sure how to troubleshoot those https://github.com/openshift/release/pull/36911#issuecomment-1507927121
@mtulio after confirmed with QE team, you can refer to FAQ-2 in https://docs.google.com/document/d/1qOxZ33lWfZRFaJC5b0U4sz3VMseMsXfTJ-1ilXgdFm4/edit to get the permission.